book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Kindly implement the `rental` method here and `book` itself should be sent as a parameter to create a new rental

Reply:
As mentioned to the previous reviewer, I did not have to create an add rental method in person. I have added an add rental method in book.rb